ast/compile.go

```diff
@@ -1277,6 +1285,26 @@ func (c *Compiler) getExports() *util.HashMap {
 	return rules
 }
 
+func (c *Compiler) checkDuplicateImports() {
```

Comment:
Many of our stages are written in a way that lets them be used in both the compiler and the query compiler.

Would we want that here? I suppose we can deal with that when the time comes... 🤔

Reply:
Right now, strict mode is only supported by the `check` command, which doesn't support queries AFAIK. But it's good to keep in mind that this might be added to other commands in the future.

Reply:
"might be added" isn't strictly true, though, as we know that some/most of the strict rules will eventually graduate to be regular rules.

Reply:
In this case, since queries don't contain imports, this implementation is fine.